frontend/components/chefugov.tsx

```diff
@@ -138,16 +138,24 @@ const ChefUgov = () => {
     );
   };
 
-setInterval(handleReward, 8e4)
+  setInterval(handleReward, 8e4);
```

Comment:
I thought we have a prettier and lint commit hook? We shouldn't have white space diff etc. pls fix

Reply:
From what I concluded, that pre-commit hook it's only on the contracts repo (it's the `husky` library) , would you like me to add it to the frontend too?

Reply:
Yeah that sounds good